### PR TITLE
Add recipe for wradlib 0.7.0

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

File: recipes/wradlib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    build:
+        - python
+        - numpy >=1.7.0
+        - scipy >=0.9
+        - matplotlib >=1.1.0
+        - h5py >=2.0.1
+        - netcdf4 >=1.0
+        - gdal
+        # The gdal package on OS X is missing a dependency on krb5
+        - krb5  # [osx]
```
Comment (pelson) [inline recipes/wradlib/meta.yaml]: More precisely, is it something like libgomp (from the top of my head)? That is certainly one I've seen on the ```netcdf-fortran``` package built by IOOS.

Comment (ocefpaf) [inline recipes/wradlib/meta.yaml]: > ```netcdf-fortran``` package built by IOOS.

`libcom_err` and it is for gdal.  (Which reminds me I have to finish #51 :flushed:)

Comment (jjhelmus) [inline recipes/wradlib/meta.yaml]: Yup, `libcom_err`. If a new gdal is build this requirement can be removed but for now it's in the recipe.

**Comment by jjhelmus** [comment]
The [CircleCI linux feedstock build](https://circleci.com/gh/conda-forge/wradlib-feedstock/1) of this package experience the same issue as the [arm_pyart feedstock](https://circleci.com/gh/conda-forge/arm_pyart-feedstock/1) when the package attempts to import matplotlib during the import test.  This is odd since the build and test works fine in the [CI of the staged-recipes](https://circleci.com/gh/conda-forge/staged-recipes/293).  The exact same packages are installed in two cases (staged-recipe and feedstock).  I'll dig more into this later today to see if I can get to the bottom of the issue.

**Comment by jjhelmus** [comment]
Linux build for this is fixed in conda-forge/wradlib-feedstock#1 and for Py-ART in conda-forge/arm_pyart-feedstock#2

**Pull request merged**